lib/sles4sap/qesap/qesapdeployment.pm

```diff
@@ -39,6 +39,8 @@ use utils qw(file_content_replace);
 use version_utils 'is_sle';
 use publiccloud::utils qw(get_credentials);
 use sles4sap::qesap::qesap_aws;
+use sles4sap::qesap::qesap_azure;
+use sles4sap::qesap::qesap_utils;
```

Comment:
Oh, I see we had `sles4sap::qesap::qesap_aws` already. Disregard my comments about the double `qesap` in package names .... my left eye will have to continue twitching :D